dynamixel_workbench_single_manager/src/dynamixel_driver.cpp

```diff
@@ -406,12 +410,19 @@ bool DynamixelDriver::initROS()
   {
     joint_st_pub_.publish(getJointState());
   };
-//  publish_joint_st_timer_ = nh_.createTimer(period,&DynamixelDriver::publishMessagesCallback,this);
+  //  publish_joint_st_timer_ = nh_.createTimer(period,&DynamixelDriver::publishMessagesCallback,this);
   publish_joint_st_timer_ = nh_.createTimer(period,publish_js_callback);
 
   // poll motor timer
   poll_motor_timer_ = nh_.createTimer(ros::Duration(1.0/polling_rate_),&DynamixelDriver::pollMotor,this);
 
+  // set name field of joint state message
+  if(!joint_.empty())
```

Comment:
Having a joint name in the name field of the joint state message isn't critical for the driver to operate, so should we really throw an error? Maybe there would be a situation where you want to actuate the motor, but don't really care about it being tied to a URDF frame